submit/templates/submit/manage_submissions.html

```diff
@@ -7,43 +7,48 @@
 
 {% block within_content %}
 <section class="columns is-marginless section">
-  <div class="column is-one-third-tablet is-one-third-desktop">
-    <div class="box has-text-centered box-new-submission">
-          <form class="form" method="POST" action="{{ url_for('ui.create_submission') }}">
-            {{ form.csrf_token }}
-            <button name="new" class="button button-is-wrappable button-create-submission is-large is-link" value="new" aria-label="Create New Submission">Start a New Submission</button>
-          </form>
-          <h3><a href="https://arxiv.org/help#submission-and-revision">Submission Help Pages</a></h3>
-
+  <div class="column box is-one-third-tablet is-one-third-desktop">
+    <div class="box-new-submission">
+      <h2>Welcome [Firstname]</h2>
```

Comment:
In this particular case you could do this:
```suggestion
      <h2>Welcome {{ user.name.forename }}</h2>
```